Justfile

```diff
@@ -93,9 +93,9 @@ check: devenv
     $BIN/black --check .
     $BIN/isort --check-only --diff .
     $BIN/flake8
-    $BIN/interrogate cohortextractor2
+    $BIN/interrogate cohortextractor
     $BIN/pyupgrade --py39-plus --keep-percent-format \
-        $(find cohortextractor2 -name "*.py" -type f) \
+        $(find cohortextractor -name "*.py" -type f) \
```

Comment:
These name changes are unexpected. Is this a mistake?

Reply:
oh no, definitely not. let me fix that now.